src/main/groovy/geb/mobile/android/AndroidUIAutomatorNonEmptyNavigator.groovy

```diff
@@ -138,8 +138,8 @@ class AndroidUIAutomatorNonEmptyNavigator extends AbstractMobileNonEmptyNavigato
                 input.click()
             }
         } else {
-            (AndroidElement)input.replaceValue(value as String)
-
+            input.clear()
```

Comment:
We had intentionally converted from the clear/sendKeys to use the replaceValue instead.  The replaceValue does a clear in the appium bootstrap command.  With the clear/sendKeys clear gets called twice adding up to an additional 2 seconds per call.  Were you seeing specific issues with replaceValue?